src/main/java/mate/academy/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,15 +1,26 @@
 package mate.academy.bookshop.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import mate.academy.bookshop.validation.Isbn;
 
 @Data
 public class CreateBookRequestDto {
     private Long id;
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @Isbn
+    @NotNull
     private String isbn;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotNull
```

Comment:
But description and coverImage are nullable in DB, right? I think they can be nullable here too.